browser/css/jsdialogs.css

```diff
@@ -1277,7 +1277,8 @@ input[type='number']:hover::-webkit-outer-spin-button {
 #mobile-wizard #wordslabel2,
 #mobile-wizard #inclspaceslabel2,
```

Comment:
don't we have already "id/name of the dialog" in the mobilewizard structure? (I indoriduced containers for separate "windows" at some point)

then we could just create a generic rule here